File: src/main/java/com/github/spartusch/RateRetrieverApplication.java
package com.github.spartusch;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.web.reactive.error.ErrorWebFluxAutoConfiguration;
import org.springframework.boot.autoconfigure.web.servlet.error.ErrorMvcAutoConfiguration;
import org.springframework.cache.annotation.EnableCaching;

@SpringBootApplication
@EnableAutoConfiguration(exclude = { ErrorWebFluxAutoConfiguration.class, ErrorMvcAutoConfiguration.class })
@EnableCaching
public class RateRetrieverApplication {

    public static void main(String[] args) {
        SpringApplication.run(RateRetrieverApplication.class, args);
    }

}

File: src/main/kotlin/com/github/spartusch/RateRetrieverApplication.kt
package com.github.spartusch

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan

@SpringBootApplication
@ConfigurationPropertiesScan
class RateRetrieverApplication

fun main(
    args: Array<String>
) {
    @Suppress("SpreadOperator")
    SpringApplication.run(RateRetrieverApplication::class.java, *args)
}
